client/hosting-overview/index.tsx

```diff
@@ -26,9 +45,12 @@ export default function () {
 		'/hosting-config/activate/:site_id',
 		siteSelection,
 		navigation,
-		redirectHomeIfIneligible,
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore
+		redirectIfCurrentUserCannot( 'manage_options' ),
```

Comment:
The original line is using the exact same method, not sure why I need to ts-ignore it now

https://github.com/Automattic/wp-calypso/blob/9794b30bbb92a2fafc08b2de08a06ff69324835e/client/my-sites/hosting/index.js#L16